### DISCUSSION TITLE
Documentation for reverse enginering

### DISCUSSION THREAD

**Discussion by perru** [liquidctl/liquidctl#336] [General]
Hello,

Is it possible to have documentation for reverse enginering Corsair ICUe gear compatible please ?

I want to add my Corsair AIO H115i CAPELLIX in the list :).

It could be usefull for all people who whant to contribute to the project.

Thank you !

**Comment by jonasmalacofilho** [comment]
Take a look at [CONTRIBUTING.md](https://github.com/liquidctl/liquidctl/blob/main/CONTRIBUTING.md).  And there are already some people working on the CAPELLIX (a.k.a. Commander Core), so also take a look at #218 and maybe join our [Discord](https://discord.gg/GyCBjQhqCd) server.

**Reply by perru** [reply]
Thank you, I dind't find it. I'll read this and check what works are done at the moment. If I can bring something usefull to the project, I will.

**Reply by jonasmalacofilho** [reply]
Great, welcome!
